spaceR/spaceR/CustomUI/UIRocketInfoCollectionViewCell.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  UIRocketInfoCollectionViewCell.swift
+//  SpaceR
+//
+//  Created by Kieran Woodrow on 2022/04/01.
+//
+
+import Foundation
+import UIKit
+
+class UIRocketInfoCollectionViewCell: UICollectionViewCell {
+    
+    @IBOutlet private weak var rocketCollectionViewTitle: UILabel!
+    @IBOutlet private weak var rocketCollectionViewInfo: UILabel!
+    private var heading = ""
+    private var info = ""
+    
+    func set(rocketTitle: String) {
+        self.rocketCollectionViewTitle.text = rocketTitle
+    }
+    
+    func set(rocketInfo: String) {
+        self.rocketCollectionViewInfo.text = rocketInfo
+    }
+    
+    func setRocketCell(viewModel: RocketInfoViewModel, indexPath: Int) {
+        setHeadings(viewModel: viewModel, indexPath: indexPath)
+        set(rocketTitle: heading)
+        set(rocketInfo: info)
+        self.layer.borderColor = UIColor(red: 1, green: 0.557, blue: 0.447, alpha: 100).cgColor
+        self.layer.borderWidth = 1
+        self.layer.cornerRadius = 10
+    }
+    
+    func setHeadings(viewModel: RocketInfoViewModel, indexPath: Int) {
+        switch indexPath {
+        case indexPath where indexPath == 0:
+            heading = "First Flight"
```

Comment:
This array of headings should live in your `ViewModel` 😄 this is coupling your code. Give me a call if you need help moving these headings to your `ViewModel`